src/dag/chunk.rs

```diff
@@ -1,8 +1,100 @@
-pub type Key = String;
+use super::meta_generated;
```

Comment:
Maybe add `::meta` to the end so we can shorten the various `meta_generated::meta` expressions in this file?

Reply:
done